cmd/relayproxy/main.go

```diff
@@ -48,37 +48,40 @@ func main() {
 	_ = f.Parse(os.Args[1:])
 
 	// Init logger
-	zapLog := log.InitLogger()
-	defer func() { _ = zapLog.Sync() }()
+	logger := log.InitLogger()
```

Comment:
@thomaspoignant any tip to satisfy this coverage? The test for main is empty, just a dummy test there, and all new code I introduced is flagged, so I believe I have to find where exactly you test this cases. Could you guide me?

Reply:
Most of the time I don't test the main function.
So I guess this is not really a problem here.

Reply:
@thomaspoignant let me know them what else needs to change to push this forward. The codecoverage isn't passing, that's why I asked the question.